scrapers/dlsite.yml

```diff
@@ -0,0 +1,100 @@
+name: DLsite
+sceneByFragment:
+  action: scrapeXPath
+  queryURL: https://www.dlsite.com/maniax/work/=/product_id/{title}.html
+  #queryURL: https://www.dlsite.com/maniax/work/=/product_id/{title}.html/?locale=en_US
+  scraper: sceneScraper
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - dlsite.com/maniax/work/
+    scraper: sceneScraper
+    queryURL: "{url}"
+sceneByName:
+  action: scrapeXPath
+  queryURL: https://www.dlsite.com/maniax/fsr/=/language/jp/sex_category%5B0%5D/male/keyword/{}/order%5B0%5D/trend/per_page/30/from/fs.header
+  scraper: sceneQueryScraper
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "{url}"
+  scraper: sceneScraper
+galleryByFragment:
+  action: scrapeXPath
+  queryURL: https://www.dlsite.com/maniax/work/=/product_id/{title}.html
+  #queryURL: https://www.dlsite.com/maniax/work/=/product_id/{title}.html/?locale=en_US
+  scraper: galleryScraper
+galleryByURL:
+  - action: scrapeXPath
+    url:
+      - dlsite.com/maniax/work/
```

Comment:
```yaml
      - dlsite.com/maniax/work/
      - dlsite.com/books/work/
 ```
Add galleries from the books section as well ? Eg `https://www.dlsite.com/books/work/=/product_id/BJ560145.html/?locale=en_US` or `https://www.dlsite.com/books/work/=/product_id/BJ560145.html`

Reply:
Sure I can include that. Might need a different galleryScraper though as there's some massaging needed around the 著者 field instead of the 作者 one that maniax use.

Reply:
Did a quick test with the URL added, didnt even notice the extra `massaging` needed...
If it needs a lot of extra work its fine as it is (no books)

Reply:
It shouldn't be too much effort. Happy to get this as feature complete as possible.